applications/API/tests/test_forms.py

```
@@ -0,0 +1,137 @@
+        self.assertTrue(form.is_valid(), form.errors.as_json())
+
+    def test_invalid_when_insult_reference_id_not_found(self):
+        """Invalid if the ref id doesn't resolve via Insult.get_by_reference_id()."""
+        data = self._base_payload(insult_reference_id="NOPE_999")
+        self.assert_form_invalid_and_error_present(data, "Invalid Insult ID")
+
+    def test_review_text_min_length_enforced_only_when_provided(self):
+        """min_length=70 should fail if provided but too short; empty is allowed."""
+        # Too short and provided -> invalid
+        data_short = self._base_payload(rationale_for_review="way too short")
+        form_short = self.assert_form_invalid_and_error_present(
```

Pylint (4.0.1)

[warning] Unused variable 'form_short'

(W0612)